Neos.Neos/webpack.styles.config.js

```diff
@@ -0,0 +1,82 @@
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const IgnoreEmitPlugin = require("ignore-emit-webpack-plugin");
+
+const stylesConfig = {
+	context: __dirname,
+	devtool: "source-map",
+	entry: {
+		Main: ["./Resources/Private/Styles/Neos.scss"],
+		Lite: ["./Resources/Private/Styles/Lite.scss"],
+		Minimal: ["./Resources/Private/Styles/Minimal.scss"],
+		Login: ["./Resources/Private/Styles/Login.scss"],
+		Error: ["./Resources/Private/Styles/Error.scss"],
+		RawContentMode: ["./Resources/Private/Styles/RawContentMode.scss"],
+		Welcome: ["./Resources/Private/Styles/Welcome.scss"],
+	},
+	output: {
+		path: __dirname + "/Resources/Public/Styles",
+		filename: "[name].js",
+	},
+	resolve: {
+		extensions: [".css", ".scss", ".sass"],
+	},
+	module: {
+		rules: [
+			{
+				test: /\.(png|gif|jpe?g|svg)$/i,
+				include: __dirname + "/Resources/Public/Images/",
+				use: [
+					{
+						loader: "url-loader",
+						options: {
+							name: "images/[name].[ext]",
```

Comment:
Good catch ... that did not end in an error because the sass already references to the right location.
<img width="536" alt="Screenshot 2021-03-27 at 20 53 43" src="https://user-images.githubusercontent.com/1014126/112733162-937b2e80-8f3e-11eb-9083-ba4e94650c64.png">